hopsworks-web/yo/app/scripts/controllers/featurestoreCtrl.js

```diff
@@ -120,6 +121,8 @@ angular.module('hopsWorksApp')
                 ProjectService.get({}, {'id': self.projectId}).$promise.then(
                     function (success) {
                         self.projectName = success.projectName;
+                        self.projectFeaturestoreName = self.projectName.toLowerCase() + "_featurestore";
+                        self.projectTrainingdatasetsName = self.projectName + "_Training_Datasets";
                         StorageService.store("projectName", self.projectName)
```

Comment:
Shouldn't this one stored as `lowerCase()` as well?

Reply:
lowercasing it would break this: https://github.com/logicalclocks/hopsworks/blob/97de848a3b9a0530b18ac6e2eb34b4a470f2b198/hopsworks-web/yo/app/scripts/controllers/newTrainingDatasetCtrl.js#L165, I guess I could store the `projectTrainingdatasetsName` in the storage and use it here but it's not obvious that projectName is lowercased so for now I vote for keeping it as is. The other place we use it is https://github.com/logicalclocks/hopsworks/blob/97de848a3b9a0530b18ac6e2eb34b4a470f2b198/hopsworks-web/yo/app/scripts/controllers/newFeaturegroupCtrl.js#L109 which I fixed too.